torch_geometric/nn/attention/sgformer.py

```diff
@@ -38,34 +40,46 @@ def __init__(
         self.k = torch.nn.Linear(channels, inner_channels, bias=qkv_bias)
         self.v = torch.nn.Linear(channels, inner_channels, bias=qkv_bias)
 
-    def forward(self, x: Tensor) -> Tensor:
+    def forward(self, x: Tensor, batch: Tensor) -> Tensor:
```

Comment:
compared to PerformerAttention’s forward function, which param design do you think is better?
https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/attention/performer.py#L147